modules/runner/src/main/java/org/apache/ignite/internal/app/IgnitionImpl.java

```diff
@@ -220,16 +218,19 @@ private static Ignite doStart(String nodeName, @Nullable String cfgContent, Path
                 new ConfigurationManager(rootKeys, cfgStorages)
             );
 
-            if (!cfgBootstrappedFromPds && cfgContent != null)
+            boolean cfgBootstrappedFromPds = locConfigurationMgr.bootstrapped();
+
+            if (!cfgBootstrappedFromPds && cfgContent != null) {
                 try {
                     locConfigurationMgr.bootstrap(cfgContent, ConfigurationType.LOCAL);
                 }
                 catch (Exception e) {
                     LOG.warn("Unable to parse user-specific configuration, default configuration will be used: {}", e.getMessage());
                 }
+            }
             else if (cfgContent != null)
                 LOG.warn("User specific configuration will be ignored, cause vault was bootstrapped with pds configuration");
-            else
```

Comment:
This condition is excessive, can it be a comment instead of the actual check?

Reply:
This place already changed.